src/EditorFeatures/CSharp/Outlining/Outliners/DocumentationCommentOutliner.cs

```diff
@@ -35,12 +36,39 @@ private static string GetBannerText(DocumentationCommentTriviaSyntax documentati
             string text;
             if (summaryElement != null)
```

Comment:
In the absence of summary tags, we just take the first line of the doc comment verbatim.